src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml

```diff
@@ -15,20 +15,20 @@
                     <h2 data-test-id="login-page-description" class="nhsuk-heading-s nhsuk-u-margin-bottom-5">Enter your details to access the full functionality of this website.</h2>
                 </div>
             </div>
-
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-one-half">
                     <form asp-controller="Account" asp-action="Login" asp-route-returnUrl="@ViewData["ReturnUrl"]" method="post">
 
-                        <input type="hidden" asp-for="ReturnUrl"/>
+                        <input type="hidden" asp-for="ReturnUrl" />
 
                         <div data-test-id="email-field">
                             <div data-test-id="text-field-input" class="">
                                 <div class="nhsuk-form-group">
                                     <label asp-for="Username" class="nhsuk-label">
                                         Email address
                                     </label>
-                                    <input asp-for="Username" class="nhsuk-input"/>
+                                    <input asp-for="Username" class="nhsuk-input" />
+                                    <span asp-validation-for="Username" class="text-danger"></span>
```

Comment:
As discussed the class used here is a bootstrap class which we currently don't reference / use within the application. I think it would be clearer here just to remove them. We should be getting a user story to decide how the login page should look with errors.